src/components/TokenPairSelector/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { TokenPairSelectorList } from 'components/TokenPairSelectorList'
+
+import ArrowWhite from 'assets/img/arrow-white.svg'
+
+const Wrapper = styled.div`
+  display: flex;
+  width: 100%;
+
+  > button {
+    width: 100%;
+    height: var(--height-bar-default);
+    background: var(--color-primary);
+    color: var(--color-text-secondary1);
+    appearance: none;
+    border: 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 1.6rem;
+    box-sizing: border-box;
+    border-bottom: 0.1rem solid var(--color-border);
+    outline: 0;
+    cursor: pointer;
+    transition: color 0.3s ease-in-out;
+
+    &:hover {
+      color: var(--color-text-primary);
+    }
+  }
+
+  > button > b {
+    font-size: 1.5rem;
+    letter-spacing: 0.03rem;
+  }
+
+  > button > i {
+    font-size: 1.2rem;
+    font-style: normal;
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: center;
+    align-items: center;
+    font-weight: var(--font-weight-medium);
+
+    &::after {
+      display: block;
+      height: 1.2rem;
+      width: 1.2rem;
+      content: '';
+      background: url(${ArrowWhite}) no-repeat center/contain;
```

Comment:
This will likely be an optional image passed as a prop to the button
We could parametrize the use of an image to the right or to the left of the button

Doesn't need to be in this PR though. Since it can easily get complicated...
we can mege soon this to develop, and then work in refactoring this into finer pieces